holmes/plugins/toolsets/json_filter_mixin.py

```
@@ -0,0 +1,126 @@
+    filter_parameters: Dict[str, ToolParameter] = {
+        "max_depth": ToolParameter(
+            description="Maximum nesting depth to return from the JSON (0 returns only top-level keys). Leave empty for full response.",
+            type="integer",
+            required=False,
+        ),
+        "jsonpath": ToolParameter(
+            description="Optional jsonpath expression to extract specific parts of the JSON. Supports extended syntax including filters, slicing, arithmetic, and sorting (e.g., '$.items[?(@.price > 10)]', '$.items[0:5]', '$.items[*].name').",
+            type="string",
+            required=False,
+        ),
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)